jx_sqlite/__init__.py

```diff
@@ -211,11 +211,15 @@ def _get(row):
     return _get
 
 
-def set_column(row, col, child, value):
+def set_column(row, col, name, child, value, header):
     if child == ".":
```

Comment:
I do not think that this method should be changed: Ideally, it should be just one line

    row[col][child]=value

but this does not work because of the `row[col]` is not initialized to a `Data` object.  I thought it would be faster to use this function, rather than initializing the `row` with `Data`, just to have them replaced with primitive values.

    row[col]["."] = value  # is the same as row[col]=value